src/cargo/ops/cargo_compile.rs

```diff
@@ -29,7 +29,7 @@
 //! [`Layout`]: crate::core::compiler::Layout
 //! ["Cargo Target"]: https://doc.rust-lang.org/nightly/cargo/reference/cargo-targets.html
 
```

Comment:
This is perpetuating our mixing of `mod.rs` / `<name>.rs` module root styles (`cargo/ops/registry.rs` is the only other one I'm away of for this style).  Personally, I feel like we should only be using one or the other

Reply:
I don't have a strong opinion on this. The reason I chose not to rename to `mod.rs` are only,

* Keep git history better. (I don't even like my refactor moving things around 😆)
* It sometimes hurts when searching files, and you get a list of `mod.rs`.

I am happy to change for consistency if people feel comfortable with mod.rs

Reply:
Which ever direction we go, I'd prefer consistency.  Since we are using `mod.rs` for most things, I lean towards us sticking with that for now.

*Granted, I'm somewhat biased in that I prefer `mod.rs`.  Despite the editor improvement to not have generic file names, I much prefer being able to select one directory to search or edit everything in a mod rather than cross them.  I also am more likely to miss that either the file or directory exists when `mod.rs` isn't used.*

Git does do a good job of keeping history across moving files.  If you do the move to the `mod.rs` in one commit and then moving content out into other files in the next commit, git will handle it.  Sometimes it will even handle it if you don;'t do it in two commits but I generally encourage moves to be in as isolated of commits as possible so people can more easily blindly trust what happened in them rather than verifying what people do in moves.

Granted, I do have the following set in my config
```gitconfig
[diff]
    renames = copies
[log]
    follow = true
```

Reply:
Oh, nice! I didn't know those git tricks. Thanks for sharing!

Updated per your suggestion :)